client/src/js/components/bhMultipleCashboxSelect.js

```diff
@@ -0,0 +1,47 @@
+angular.module('bhima.components')
+  .component('bhMultipleCashboxSelect', {
+    templateUrl : 'modules/templates/bhMultipleCashBoxSelect.tmpl.html',
+    controller  : MultipleCashboxSelectController,
+    bindings    : {
+      onChange : '&',
+      cashboxesIds : '<?',
+      label : '@?',
+      required : '<?',
+      validationTrigger : '<?',
+    },
+  });
+
+MultipleCashboxSelectController.$inject = [
+  'CashboxService', 'NotifyService',
+];
+
+/**
+ * Multiple Cashbox Selection Component
+ *
+ */
+function MultipleCashboxSelectController(Cashbox, Notify) {
+  var $ctrl = this;
+
+  $ctrl.$onInit = function onInit() {
+    // label to display
+    $ctrl.label = $ctrl.label || 'FORM.LABELS.CASHBOX';
+
+    // fired when a Cashbox has been selected or removed from the list
+    $ctrl.onChange = $ctrl.onChange || angular.noop;
```

Comment:
Since this function is required in your bindings (`&`), there isn't any need to alias it with a `noop` function.